conf/repos.yaml.template

```diff
@@ -33,6 +33,14 @@ REPOS:
     RHEL7: replace-with-rhel7-http-link
     RHEL8: replace-with-rhel8-http-link
     RHEL9: replace-with-rhel9-http-link
+  # Satellite's utils repos, and it works for 7.0 and onwards
```

Comment:
This is a really ugly crossover in validation of the yamls, since these fields were introduced by sat6-upgrade. Not directly related to this PR or needing to be fixed in this PR. But the fact that GHA will fail for robottelo PRs because satellite6-upgrade added some fields to its repos.yaml is broken.

Reply:
Yeah! Agree but the checks were failing for upgrade scenarios collections where it's looking for setting in robottelo's confs.